server/src/test/scala/actors/ShiftsActorSpec.scala

```diff
@@ -28,75 +27,250 @@ class ShiftsActorSpec extends CrunchTestLike with ImplicitSender {
 
   import StaffAssignmentGenerator._
 
+  def assertExpectedResponse(expectedShifts: ShiftAssignments): Unit = {
+    val response = expectMsgType[ShiftAssignments]
+    val sortedResponse = response.assignments.sortBy(_.start)
+    val sortedExpectedShifts = expectedShifts.assignments.sortBy(_.start)
+    assert(sortedResponse == sortedExpectedShifts)
+  }
+
   "Shifts actor" should {
     "remember a shift staff assignment added before a shutdown" in {
       val startTime = SDate(s"2017-01-01T07:00").millisSinceEpoch
-      val endTime = SDate(s"2017-01-01T15:00").millisSinceEpoch
+      val endTime = SDate(s"2017-01-01T12:00").millisSinceEpoch
       val shifts = ShiftAssignments(Seq(StaffAssignment("Morning", T1, startTime, endTime, 10, None)))
 
+      val expectedShifts = ShiftAssignments(Seq(
```

Comment:
Can you reduce the length of the shift to 30 minutes so we only need to check there are 2x 15 minute shifts created - it'll be a much shorter amount of code